glibc >= 2.31 dropped stime() from the link interface; use clock_settime()
as busybox itself did later.
--- a/coreutils/date.c	2026-09-15 00:35:01.299224641 -0500
+++ b/coreutils/date.c	2026-09-15 00:35:01.307224639 -0500
@@ -322,7 +322,7 @@
 		maybe_set_utc(opt);
 
 		/* if setting time, set it */
-		if ((opt & OPT_SET) && stime(&ts.tv_sec) < 0) {
+		if ((opt & OPT_SET) && clock_settime(CLOCK_REALTIME, &ts) < 0) {
 			bb_perror_msg("can't set date");
 		}
 	}
--- a/util-linux/rdate.c	2026-09-15 00:35:01.303224640 -0500
+++ b/util-linux/rdate.c	2026-09-15 00:35:01.315224638 -0500
@@ -98,7 +98,10 @@
 		if (time(NULL) == remote_time)
 			bb_error_msg("current time matches remote time");
 		else
-			if (stime(&remote_time) < 0)
+			struct timespec ts;
+			ts.tv_sec = remote_time;
+			ts.tv_nsec = 0;
+			if (clock_settime(CLOCK_REALTIME, &ts) < 0)
 				bb_perror_msg_and_die("can't set time of day");
 	}
 
